transport_server.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/pkg/errors"
+	"github.com/status-im/status-go/node"
+)
+
+type server struct {
+	node *node.StatusNode
```

Comment:
Can we make it an interface as well? I would like to avoid importing `github.com/status-im/status-go/node`. There were some problems with that. Generally, we need to figure out some better way of passing geth server here in order to avoid dependency on status-go completely.

Reply:
i thought we want to make `protocol` module independent from status-go, and move it eventually to another repo.
i can hide status node behind interface for sure, i just don't see why we cannot import status-go in main module of the status-console-client

Reply:
Ah sorry, it's in the main module. I thought it's in `protocol`. All good.